.github/workflows/bund_app_release_restricted.yml

```diff
@@ -7,35 +7,13 @@ permissions:
 
 jobs:
   column_1_restricted:
-    runs-on: ubuntu-latest
-    env:
-      CIRRUS_CI_API_TOKEN: ${{ secrets.CIRRUS_CI_API_TOKEN }}
-    steps:
-      - uses: actions/checkout@v3
-        with:
-          fetch-depth: 0
-      - name: Set up Ruby
-        uses: ruby/setup-ruby@v1
-        with:
-          ruby-version: '3.1'
-      - name: Install dependencies
-        run: bundle install
-      - name: Trigger build
-        run: bundle exec fastlane trigger_api_build_cirrus_ci task:"AppStore C1 Restricted"
+    uses: ./.github/workflows/app_release.yml
```

Comment:
does this solve the issue with one would be cancelled automatically by the other?

Reply:
yes